metallb/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - markmc
```

Comment:
markmc might find himself surprised to be in here, though I'm not against it.  :-)

Maybe you meant markdgray ?

Reply:
Thank you for pointing it out! fixed.